commander/src/bootstrapping/templates/lisk-template-ts/templates/init/src/commands/genesis-block/create.ts

```diff
@@ -4,7 +4,8 @@ import { getApplication } from '../../app/app';
 import { registerModules } from '../../app/modules';
 
 export class GenesisBlockCommand extends BaseGenesisBlockCommand {
-		public getApplication(
+	// eslint-disable-next-line class-methods-use-this
```

Comment:
I think this rule was removed in the lisk-base. Are you sure this has error?

Reply:
Its not an error, its causing warning.

Reply:
Is it using correct lisk-base version? https://github.com/LiskHQ/eslint-config-lisk-base/blob/master/ts.js#L15
it looks it should be off

Reply:
Yes the version itself is correct. 

```
		"eslint-config-lisk-base": "2.0.0",
```

Reply:
@shuse2 Found the issue, fixing it now. We were using wrong eslint config.